client/src/pages/SignUp.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
+import Joi from 'joi';
+
+import {
+  Avatar,
+  Button,
+  CssBaseline,
+  TextField,
+  Link,
+  Grid,
+  Box,
+  Typography,
+  Container,
+} from '@mui/material';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import axios from 'axios';
+import BasicSelect from '../components';
+
+const Copyright = (props: any) => {
```

Comment:
Move this to its own file in the components directory under a signup folder and import it here. This improves redability.